src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GroupFeedback/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage, intlShape } from '@edx/frontend-platform/i18n';
+import SettingsOption from '../../SettingsOption';
+import messages from './messages';
+import { groupFeedbackCardHooks, groupFeedbackRowHooks } from './hooks';
+import GroupFeedbackRow from './GroupFeedbackRow';
+import Button from '../../../../../../../sharedComponents/Button';
+
+export const GroupFeedbackCard = ({
+  groupFeedbacks,
+  updateSettings,
+  answers,
+  // inject
+  intl,
+}) => {
+  const { summary, handleAdd } = groupFeedbackCardHooks(groupFeedbacks, updateSettings);
+  return (
+    <SettingsOption
+      title={intl.formatMessage(messages.groupFeedbackSettingTitle)}
+      summary={intl.formatMessage(summary.message, { ...summary.values })}
```

Comment:
The mock ups include the answers that are associated with the feedback, do we still want that for the beta?
<img width="459" alt="Screenshot 2023-02-07 at 11 43 27 AM" src="https://user-images.githubusercontent.com/42981026/217308206-e5dd62d6-4fde-4b37-b0c0-e3e69c042b43.png">